app/web_hub.go

```diff
@@ -539,13 +539,20 @@ func (h *Hub) Start() {
 						}
 					}
 				}
-				if msg.GetBroadcast().UserId != "" {
+
+				if connID := msg.GetBroadcast().ConnectionId; connID != "" {
```

Comment:
I am wondering what is the difference between adding the connectionId as part of the connIndex versus adding the subscription topic to the index. Any reason to do this, but not do the other? Both seem to reduce iteration time, and both would be accepted/rejected anyways in the web_conn goroutine.

Reply:
I think the main difference is semantic. This is meant to be a lower level addition than topics. We are extending the existing API for comfort but technically we could go one step further and implement a `WebSocketSend()` method and allow the plugin side to send whatever data, not constrained by the broadcast struct format. 

The case I am trying to cover is one where I don't necessarily want clients to subscribe to anything but at the same time I need full control (server-side) on which connection the data is targeting.